src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java

```diff
@@ -50,10 +50,10 @@
      *
      * This argument allows you to set the GQ boundaries. HC expects a list of multiple GQ threshold values. To pass
      * multiple values, you provide them one by one with the argument, as in `-GQB 10 -GQB 20 -GQB 30` and so on. Note
-     * that GQ values are capped at 99 in the GATK.
+     * that GQ values are capped at 99 in the GATK, so values must be integers in the range [1, 99].
```

Comment:
I think the max may be 100 if you want to capture 99's in a band with anything else.